### DISCUSSION TITLE
Apply leading to all font-size

### DISCUSSION THREAD

**Discussion by hexionaut** [tailwindlabs/tailwindcss#10122] [Help]
Hi,

As you may know, when we use a `text-sm` for example then a `md:text-xl` and say we want  `leading-snug` we have to apply it for both screen sizes as the `text-` classes already have line-height defined. 

Do we have a better way than applying leading for all screen sizes ?

Thanks for you help.

**Comment by adamwathan** [comment]
Hey! If you don't want any line-height value applied when setting a font size, you can customize your `fontSize` configuration to not include line-heights:

https://tailwindcss.com/docs/font-size#customizing-your-theme

Generally though it should be very rare that you want to use the same line-height (even a proportional line-height like `snug`) across different font sizes. In typography that's just not generally desirable — you always want a tighter line-height for bigger text and a looser line-height for smaller text so even if you don't want to use the default values for the font-size, you're still going to want to specify _something_ every time you change the font size.
